freqtrade/exchange/exchange.py

```diff
@@ -1038,10 +1038,10 @@ def get_rate(self, pair: str, refresh: bool, side: str) -> float:
                 )
                 raise PricingError from e
 
-            logger.info(f"{name} price from orderbook {conf_strategy['price_side'].capitalize()}"
-                        f"side - top {order_book_top} order book {side} rate {rate:.8f}")
+            #logger.info(f"{name} price from orderbook {conf_strategy['price_side'].capitalize()}"
+            #            f"side - top {order_book_top} order book {side} rate {rate:.8f}")
         else:
-            logger.info(f"Using Last {conf_strategy['price_side'].capitalize()} / Last Price")
+            #logger.info(f"Using Last {conf_strategy['price_side'].capitalize()} / Last Price")
```

Comment:
This will be a side-effect of #5284 - which combined buy and sell rate gathering.
While it's ok on buy to echo this "whenever" - the log-level on the sell side should be debug to avoid being verbose.

I'd change the log-level to debug instead of removing this - as there is (potentially) information there that can be enabled selectively.

Reply:
Yep-yep-yep! As long as these don't show up at "normal" log levels, I'm totally fine with that. Thank you!